src/Tizen.NUI.Components/Theme/DefaultTheme.cs

```diff
@@ -1,4 +1,4 @@
-/*
+﻿/*
```

Comment:
if you don't mind, could you remove the warnings shown below?
you can see the build warnings in VisualStudio or this PR reviewing page.
thank you.

Reply:
The warnings below are not related this commit. 
To keep atomic commits, i think we can handle this warning in another patch.